src/opam.export

```diff
@@ -235,3 +238,36 @@ installed: [
   "zarith_stubs_js.v0.14.1"
   "zed.3.1.0"
 ]
+pinned: [
```

Comment:
random thought looking at this file. I was investigating adding pins here instead of our setup script, but modifying this file by hand seems hard (no documentation on how to do it) and error prone (as it is supposed to be a machine generated file).

It would be nice to be able to list all the dependencies, all the pins, and the OCaml version we use, in a single `.toml` or `.json` file

Reply:
Yeah, I'd really like to add all pins here in the long term, and potentially get rid of submodules. I'd argue that modifying this file by hand is feasible (this is what I've been doing thus far), but we can talk about generating it from something else.

We can have that discussion in this issue: https://github.com/MinaProtocol/mina/issues/10982